relevanceai/vector_tools/plot_txt_gist.py

```diff
@@ -0,0 +1,202 @@
+import string
+import nltk
+from collections import Counter
+from typing import List
+from nltk.corpus import stopwords
+
+class base_text_processing():    # ToDo: move to DocUtils ??? If yes, where? ALSO, Requirement: nltk.download('stopwords')
```

Comment:
stylistically - python uses Upper case titling for classes. 

"BaseTextProcessing"

Reply:
Move it into a new folder called data_tools